tests/Refinery/String/Transformation/EstimatedReadingTimeTest.php

```diff
@@ -1,10 +1,10 @@
 <?php declare(strict_types=1);
 /* Copyright (c) 1998-2019 ILIAS open source, Extended GPL, see docs/LICENSE */
-
 namespace ILIAS\Tests\Refinery\String;
 
 use ILIAS\Refinery\Factory;
 use PHPUnit\Framework\TestCase;
+require_once 'libs/composer/vendor/autoload.php';
```

Comment:
I wonder why we need this. What command do you use to run the tests? This looks like something we should solve in general.

Reply:
Argggghh

Reply:
Thx @klees , good catch. I just removed the line.